File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.py
class Solution(object):
    def reverseDegree(self, s):
        """
        :type s: str
        :rtype: int
        """
        total = 0

        for i in range(len(s)):
            total += (ord('z') - ord(s[i]) + 1) * (i + 1)

        return total

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.cpp
#include <string>

class Solution {
public:
    int reverseDegree(std::string s) {
        int ans = 0;
        for (int i = 0; i < s.length(); ++i) {
            ans += (26 - (s[i] - 'a')) * (i + 1);
        }
        return ans;
    }
};